### PR TITLE
Add page on data retention.

### PR DESCRIPTION
**Pull request by Crell**

### CODE REVIEW

File: src/administration/data-retention.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Enterprise backups
+
+Platform.sh takes a backup of Platform.sh Enterprise environments ever 6 hours.  These snapshots are retained for _2 weeks_.
+
+## Tombstone backups
+
+When a project is deleted Platform.sh takes a final snapshot of active environments as well as the Git repository holding user code.  This final snapshot is to allow Platform.sh to recover a recently-deleted project in case of accident.
+
+These "tombstone" backups are retained for X days.
```
Comment (Crell) [inline src/administration/data-retention.md]: @ralt Please confirm how long this is.

File: src/administration/data-retention.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Application logs
+
+Application logs on each customer environment are retained with the environment.  Individual log files are truncated at 10 MB, regardless of their age.  See the [accessing logs](/development/logs.md) page for instructions on how to access them.
+
+When an environment is deleted its application logs are deleted as well.
+
+## Snapshots
+
+Snapshots on Platform.sh Professional are retained for at least _7 days_.  They will be purged after X days.
```
Comment (Crell) [inline src/administration/data-retention.md]: @ralt @rinchen When do we want to say snapshots get purged for reals?

Comment (ralt) [reply src/administration/data-retention.md]: "between 7 days and 6 months"

File: src/administration/data-retention.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Enterprise backups
+
+Platform.sh takes a backup of Platform.sh Enterprise environments ever 6 hours.  These snapshots are retained for _2 weeks_.
```
Comment (ralt) [inline src/administration/data-retention.md]: Every*

File: src/administration/data-retention.md (hunk 1 above)
Comment (ralt) [reply src/administration/data-retention.md]: Same as snapshots.

**Pull request merged**